apps/codexgraph_agent/pages/components/setting.py

```diff
@@ -123,7 +123,7 @@ def setting(page_name, path='CG_conversation'):
 
             with col2:
                 # Model selectbox
-                options = ['deepseek-coder', 'gpt-4o']
+                options = ['deepseek-coder', 'gpt-4o', "ollama"]
```

Comment:
this looks wierd....ollama is a model serving/inferencing framework, it is not a model in itself.

all models integratred via open-ai api compatible mode, should be done in the same way, providing only:

base_url, api key, and model name 

ollama or vllm or lmdeploy...should not be relevant.